driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/BatchCursorPublisher.java

```diff
@@ -112,6 +112,7 @@ public Publisher<T> first() {
                                     sink.success(results.get(0));
                                 }
                             })
+                            .contextWrite(sink.contextView())
```

Comment:
This is an oddity because it just propagates the context view to the inner publisher. 

But the context didn't break the chain as the outer publisher (the batch cursor publisher) had already been fixed. So the tests don't actually fail if this change is removed.